pagopa/pn-helpdesk-fe
# fix: fixed typescript and dependencies errors

## Short description
Fixed some typescript and dependencies errors that was causing an error on build. 

## List of changes proposed in this pull request
- Added missing dependencies
- Update dependencies
- Fix Typescript errors

## How to test
Run an `npm install` and start webapp with `npm start`. You should not see compiler errors
